Gems/ScriptCanvas/Code/Editor/View/Windows/MainWindow.h

```diff
@@ -769,7 +769,7 @@ namespace ScriptCanvasEditor
         void ClearStaleSaves();
         bool IsRecentSave(const SourceHandle& handle) const;
         void MarkRecentSave(const SourceHandle& handle);
-        AZStd::recursive_mutex m_mutex; 
-        AZStd::unordered_map <AZStd::string, AZStd::chrono::system_clock::time_point> m_saves;
+        AZStd::recursive_mutex m_mutex;
+        AZStd::unordered_map <AZStd::string, AZStd::chrono::steady_clock::time_point> m_saves;
```

Comment:
This isn't cached and re-used across sessions of the tool, is it?

If so, steady_clock doesn't seem useful for storing times between rebooting a machine. If not, then it's fine.

Reply:
This is used to update the file tracker state of an open script canvas file that has been [saved in the previous 2 seconds](https://github.com/o3de/o3de/blob/development/Gems/ScriptCanvas/Code/Editor/View/Windows/MainWindow.cpp#L4438-L4447).
The file tracker state is used to indicate whether a file has been [modified or saved](https://github.com/o3de/o3de/blob/development/Gems/ScriptCanvas/Code/Editor/View/Windows/MainWindow.cpp#L1049-L1060) and update the taskbar with that information. Such foo.scriptcanvas*, when it has been modified, but not saved yet.